File: VB/NetCore/WhatIsMyBrowser.CommonTypesCoreVB/UserAgentType.vb
Imports Newtonsoft.Json

Public Class UserAgentType
    <JsonProperty("user_agent")>
    Public Property UserAgent As String

    <JsonProperty("user_agent_meta_data")>
    Public Property UserAgentMetaData As UserAgentMetaData

    <JsonProperty("parse")>
    Public Property Parse As UserAgentSearchResponseParse
End Class

File: CS/NET40/WhatIsMyBrowser.CommonTypes/UserAgentType.cs
using Newtonsoft.Json;

namespace WhatIsMyBrowser.CommonTypes
{
    public class UserAgentType
    {
        [JsonProperty("user_agent")]
        public string UserAgent { get; set; }

        [JsonProperty("user_agent_meta_data")]
        public UserAgentMetaData UserAgentMetaData { get; set; }

        [JsonProperty("parse")]
        public UserAgentSearchResponseParse Parse { get; set; }
    }
}